configuration/s02_pipeline_config.json

```diff
@@ -0,0 +1,97 @@
+
+    {"RapidProKey": "Gender (Value) - imaqal_demog", "PipelineKey": "gender_raw"},
+    {"RapidProKey": "Gender (Time) - imaqal_demog", "PipelineKey": "gender_time"},
+    {"RapidProKey": "Age (Value) - imaqal_demog", "PipelineKey": "age_raw"},
+    {"RapidProKey": "Age (Time) - imaqal_demog", "PipelineKey": "age_time"},
+    {"RapidProKey": "Recently_Displaced (Value) - imaqal_demog", "PipelineKey": "recently_displaced_raw"},
+    {"RapidProKey": "Recently_Displaced (Time) - imaqal_demog", "PipelineKey": "recently_displaced_time"},
+    {"RapidProKey": "Household_Language (Value) - imaqal_demog", "PipelineKey": "household_language_raw"},
+    {"RapidProKey": "Household_Language (Time) - imaqal_demog", "PipelineKey": "household_language_time"},
+    {"RapidProKey": "District (Value) - imaqal_demog", "PipelineKey": "location_raw"},
+    {"RapidProKey": "District (Time) - imaqal_demog", "PipelineKey": "location_time"},
+
+    {"RapidProKey": "Gender (Value) - imaqal_covid19_demog", "PipelineKey": "gender_raw"},
+    {"RapidProKey": "Gender (Time) - imaqal_covid19_demog", "PipelineKey": "gender_time"},
+    {"RapidProKey": "Age (Value) - imaqal_covid19_demog", "PipelineKey": "age_raw"},
+    {"RapidProKey": "Age (Time) - imaqal_covid19_demog", "PipelineKey": "age_time"},
+    {"RapidProKey": "Recently_Displaced (Value) - imaqal_covid19_demog", "PipelineKey": "recently_displaced_raw"},
+    {"RapidProKey": "Recently_Displaced (Time) - imaqal_covid19_demog", "PipelineKey": "recently_displaced_time"},
+    {"RapidProKey": "Household_Language (Value) - imaqal_covid19_demog", "PipelineKey": "household_language_raw"},
+    {"RapidProKey": "Household_Language (Time) - imaqal_covid19_demog", "PipelineKey": "household_language_time"},
+    {"RapidProKey": "District (Value) - imaqal_covid19_demog", "PipelineKey": "location_raw"},
+    {"RapidProKey": "District (Time) - imaqal_covid19_demog", "PipelineKey": "location_time"}
+  ],
+  "ProjectStartDate": "2021-04-23T08:00:00+03:00",
+  "ProjectEndDate": "2100-01-01T00:00:00+03:00",
+  "FilterTestMessages": true,
+  "MoveWSMessages": true,
+  "AutomatedAnalysis": {
+    "GenerateRegionThemeDistributionMaps": false,
+    "GenerateDistrictThemeDistributionMaps": false,
+    "GenerateMogadishuThemeDistributionMaps": false
+  },
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "imaqal_covid19_analysis_outputs/s02/production.csv",
+    "MessagesUploadPath": "imaqal_covid19_analysis_outputs/s02/messages.csv",
+    "IndividualsUploadPath": "imaqal_covid19_analysis_outputs/s02/individuals.csv",
+    "AutomatedAnalysisDir": "imaqal_covid19_analysis_outputs/s02/automated_analysis"
+  },
+  "MemoryProfileUploadBucket":"gs://avf-pipeline-logs-performance-nearline",
+  "DataArchiveUploadBucket": "gs://pipeline-execution-backup-archive",
+  "BucketDirPath": "2021/IMAQAL_COVID19/"
```

Comment:
This might need to be 2020 given that's when this project started, not sure which makes more sense

Reply:
I think we use the year the series started instead of the project as we do on Wusc.